src/spatio_temporal_voxel_layer.cpp

```diff
@@ -112,18 +114,17 @@ void SpatioTemporalVoxelLayer::onInitialize(void)
     default_value_ = costmap_2d::FREE_SPACE;
   }
 
-  if (_publish_voxels)
-  {
-    _voxel_pub = nh.advertise<sensor_msgs::PointCloud2>("voxel_grid", 1);
-  }
+  _voxel_pub = nh.advertise<sensor_msgs::PointCloud2>("voxel_grid", 1);
```

Comment:
Please add the if statement back in, I dont want to create an unused publisher. This may show up on a rostopic list and confuse folks.